### ISSUE TITLE
output backend for Linux

### ISSUE THREAD

**Issue by Timtam** [Timtam/pyAGE#5]
there currently is no speech available under Linux. An output backend should be provided to work under Linux to support screen reader like speech feedback. I know that there was speech dispatcher available under Linux some years ago, but I don't know what the current options would be. Maybe embedding eSpeak directly would be an option.
